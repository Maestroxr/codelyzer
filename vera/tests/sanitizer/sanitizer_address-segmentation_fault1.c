#include<stdio.h> 
#include<stdlib.h> 
#include<string.h> 
void func(char ** argv) 
{ 
	char arr[ 2 ] = { 0 }; 
	strcpy(arr, argv[ 1 ]); 
	return; 
} 
int main(int argc, char *argv[]) 
{ 
	func(argv); 
	return 0; 
}