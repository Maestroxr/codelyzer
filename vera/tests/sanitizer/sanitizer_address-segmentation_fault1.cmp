C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault1.c:8: Uninitialized identifier 'p' of type char*.
C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault1.c:10: Spacing for identifier/literal in operator [], supposed to be '[ 10 ]'.
C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault1.c:10: Memory:[C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault1]: Segmentation fault, suspect variables: [p] at:
		#0 sanitizer_address-segmentation_fault1.c:10 in function 'main'
