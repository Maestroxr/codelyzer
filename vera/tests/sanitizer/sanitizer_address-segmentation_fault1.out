C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault1.c:8: Uninitialized identifier 'p' of type char*.
C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault1.c:10: Memory:[C:\Work\AutoAssessment\codelyzer\vera\tests\sanitizer\sanitizer_address-segmentation_fault1]: Segmentation fault, suspect variables: ['p'] at:
		sanitizer_address-segmentation_fault1.c:10 in function 'main'
!MemoryEnd!
